src/components/text-input/text-input.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./text-input.scss";
+
+/**
+ * TextInput reusable component - generates input with label and description
```

Comment:
A (minor) comment: TextInput can be associated with  <input type="text" /> even though it'll include other inputs as well, like type="email". While I do realize these types(email, tel, number and so on) are still text, I think renaming into something more generic might be good semantic-wise.

Reply:
Will be renamed to FormInput